src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -758,6 +759,17 @@ public class SimpleEvents {
 				)
 				.since("2.7");
 
+		if (Skript.classExists("org.bukkit.event.command.UnknownCommandEvent")) {
+			Skript.registerEvent("Unknown Command", SimpleEvent.class, UnknownCommandEvent.class, "unknown command")
+				.description("Called when an unknown command is used.")
+				.examples(
+					"on unknown command:",
+						"\tset unknown command message to \"Hey, this command does not exist.\""
+				)
+				.since("INSERT VERSION")
+				.requiredPlugins("Paper");
```

Comment:
the package is bukkit, not sure this requires paper

Reply:
thats there already tf

Reply:
![image](https://github.com/user-attachments/assets/674a3eb3-42b9-40b6-a9c6-8ceb59325626)

Reply:
nvm this is actually for paper